vscode-extension/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Development Extension",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
+      "preLaunchTask": "ts-build",
+      "env": {
+        "RELAY_BINARY_PATH": "/path/to/your/binary",
```

Comment:
I'm guessing we should omit this?

Reply:
Confirmed, this causes the extension to crash (unless you know to edit it) but removing it allowed it to work!
<img width="1260" alt="Screen Shot 2022-04-04 at 7 49 18 PM" src="https://user-images.githubusercontent.com/162735/161669574-e1909825-2509-44a9-8e53-acf8aea7b967.png">

Reply:
this is temporary and probably helped you run it locally? probably fine to keep it until we get proper vscode config setup.

Reply:
Once I removed it my server started correctly. I don't think we need it.

Reply:
Oh, I see what you're saying. This lets you use a locally compiled version of the binary. I have ` "relay-compiler": "0.0.0-main-7a27722d",` in my package.json, so it's using a recent enough version that the compiler supports the LSP.

Maybe leave it but comment it out? (I think VSCode's JSON parser supports comments?)